query-engine/connector-test-kit-rs/query-tests-setup/src/connector_tag/js/external_process.rs

```diff
@@ -140,10 +143,11 @@ fn start_rpc_thread(mut receiver: mpsc::Receiver<ReqImpl>) -> Result<()> {
                                                 // The other end may be dropped if the whole
                                                 // request future was dropped and not polled to
                                                 // completion, so we ignore send errors here.
-                                                _ = sender.send(success.result);
+                                                _ = sender.send(Ok(success.result));
                                             }
                                             jsonrpc_core::Output::Failure(err) => {
-                                                panic!("error response from jsonrpc: {err:?}")
+                                                tracing::error!("error response from jsonrpc: {err:?}");
```

Comment:
Otherwise, `panic!` kills test process and tests won't run past the first unimplemented method.